ruma-events-macros/src/event_enum.rs

```diff
@@ -202,6 +206,134 @@ fn marker_traits(ident: &Ident) -> TokenStream {
     }
 }
 
+fn accessor_methods(ident: &Ident, variants: &[Ident]) -> syn::Result<TokenStream> {
+    let any_content = ident.to_string().replace("Stub", "").replace("Stripped", "");
+
+    let content_enum = Ident::new(&format!("{}Content", any_content), ident.span());
+
+    let origin_server_ts = if ident == "AnyBasicEvent"
+        || ident == "AnyEphemeralRoomEvent"
+        || ident == "AnyEphemeralRoomEventStub"
+        || ident == "AnyToDeviceEvent"
+        || ident == "AnyStrippedStateEventStub"
+    {
+        TokenStream::new()
+    } else {
+        quote! {
```

Comment:
There should be a helper function that generates an accessor method given the field name, the field type, and the enum's variants.